src/components/Poster.js

```diff
@@ -8,7 +8,7 @@ const Poster = ({ movie }) => {
         pathname:`/movie-details/${movie.title.replace(/\s+/g, '')}`,
         state: {movie}
       }}>
-      <img src={movie.poster_path} className='poster'/>
+      <img src={movie.poster_path} alt={movie.title} className='poster'/>
```

Comment:
Great catch on providing an alt for the images! Should we maybe do `${movie.title} poster` to be more specific?